l2.py

```diff
@@ -0,0 +1,234 @@
+        """Directory storing chain data."""
+
+        self.password_path = f"{self.data_dir}/password"
+        """Path to file storing the password for the signer key."""
+
+        self.password = "l2_devnet_password"
+        """Password to use to secure the signer key."""
+
+        self.tmp_signer_key_path = f"{self.data_dir}/block-signer-key"
+        """Path to file storing the signer key during the initial import."""
+
+        self.signer_address = "0xca062b0fd91172d89bcd4bb084ac4e21972cc467"
+        """Address of the block signer."""
+
+        self.signer_private_key = "3e4bde571b86929bf08e2aaad9a6a1882664cd5e65b96fff7d03e1c4e6dfa15c"
+        """Private key of the block signer."""
+
+        genesis = lib.read_json_file(paths.l2_genesis_path)
+        self.chain_id = genesis["config"]["chainId"]
+
+        self.jwt_secret_path = paths.ops_bedrock_dir + "/test-jwt-secret.txt"
+        """Path for Jason Web Token secret, used for devnet op-geth rpc auth."""
+
+        # For the following values, allow environment override for now, to follow the original.
+        # In due time, remove that as we provide our own way to customize.
+
+        self.verbosity = os.environ.get("GETH_VERBOSITY", 3)
+        """Geth verbosity level (from 0 to 5, see geth --help)."""
+
+        self.rpc_port = os.environ.get("RPC_PORT", 9545)
+        """Port to use for the http-based JSON-RPC server."""
+
+        self.ws_port = os.environ.get("WS_PORT", 9546)
+        """Port to use for the WebSocket-based JSON_RPC server."""
+
+
+####################################################################################################
+
+def start_devnet_l2_node(paths: OPPaths):
+    """
+    Spin the devnet op-geth node, then wait for it to be ready.
+    """
+
+    cfg = DevnetL2Config(DEVNET_L2_DATA_DIR, paths)
+
+    # Make sure the port isn't occupied yet.
+    # Necessary on MacOS that easily allows two processes to bind to the same port.
+    running = True
+    try:
+        lib.wait("127.0.0.1", cfg.rpc_port, retries=1)
+    except Exception:
+        running = False
+    if running:
+        raise Exception("Couldn't start op-geth node: server already running at localhost:9545")
+
+    # Create geth db if it doesn't exist.
+    os.makedirs(DEVNET_L2_DATA_DIR, exist_ok=True)
+
+    if not os.path.exists(cfg.keystore_dir):
+        # Initial account setup
+        print(f"Directory '{cfg.keystore_dir}' missing, running account import.")
+        with open(cfg.password_path, "w") as f:
+            f.write(cfg.password)
+        with open(cfg.tmp_signer_key_path, "w") as f:
+            f.write(cfg.signer_private_key.replace("0x", ""))
+        lib.run(
+            "importing signing keys",
+            ["op-geth", "account", "import",
+             f"--datadir={cfg.data_dir}",
+             f"--password={cfg.password_path}",
+             cfg.tmp_signer_key_path])
+        os.remove(f"{cfg.data_dir}/block-signer-key")
```

Comment:
Apparently the whole password thing is not needed for the L2 side? That would mean the signing is done by the op-node.

Could you test if that is indeed the case? (If not, it means it was changed after the old commit we're using.)